filebeat/_meta/fields.common.yml

```diff
@@ -114,3 +114,20 @@
         log events this is when the log line was read by Filebeat. In comparison
         @timestamp is the processed timestamp from the log line. If both are identical
         only @timestamp should be used.
+
+    - name: http.response.status_code
+      type: long
+      description: >
+        HTTP response status_code.
+      example: 404
+
+    - name: http.response.time
```

Comment:
I'm not happy with the naming of this field as it's more the duration of the request and does not directly belong to the response?

Reply:
This is probably what we have in heartbeat under `http.rtt.total`: https://github.com/elastic/beats/blob/master/heartbeat/monitors/active/http/_meta/fields.yml#L85

Reply:
Any suggestions?

Reply:
I like the suggestion, changed to elapsed_time